docs/types/mapping-types.rst

```diff
@@ -41,7 +41,7 @@ contract that returns the value at the specified address.
 
 ::
 
-    pragma solidity >=0.4.0 <0.7.0;
+    pragma solidity >0.4.0 <0.7.0;
```

Comment:
Did this not work with 0.4.0?

Reply:
Script will run against 0.4.11 as the first realesed version on github
I am not sure for particular example if it works with 0.4.0

Reply:
Leaving with what it was before